chroma-agent/chroma_agent/action_plugins/manage_targets.py

```diff
@@ -115,7 +115,13 @@ def get_resource_locations():
     """Parse `crm_mon -1` to identify where (if anywhere)
        resources (i.e. targets) are running."""
 
-    rc, lines_text, stderr = AgentShell.run_old(["crm_mon", "-1", "-r"])
+    try:
+        rc, lines_text, stderr = AgentShell.run_old(["crm_mon", "-1", "-r"])
+    except OSError, e:
+        # ENOENT is fine here.  Pacemaker might not be installed yet.
+        if e.errno != errno.ENOENT:
```

Comment:
Is this the only case where this can happen?

I.E. what if the file is removed somehow *after* it's installed?